forms-flow-api/src/formsflow_api/models/process.py

```diff
@@ -136,3 +137,13 @@ def get_latest_version(cls, process_name):
         )
 
         return query
+
+    @classmethod
+    def fetch_histories_by_process_name(cls, process_name: str) -> List["Process"]:
+        """Fetch all versions (histories) of a process by process_name."""
+        assert process_name is not None
+        query = (

```

Comment:
Shouldn't this add tenant authorization, or is it added somewere else ?

Reply:
already included in auth_query